config/api.yml

```diff
@@ -1272,6 +1272,24 @@
       :delete:
       - :name: delete
         :identifier: chargeback_rates_delete
+  :regions:
+    :description: Regions
+    :identifier: region
+    :options:
+    - :collection
+    :verbs: *gp
```

Comment:
Since you're adding post, can one update a region with the default implementation?